docs/specs/xref.yml

```diff
@@ -2027,3 +2027,61 @@ outputs:
         {"uid": "c", "href": "https://docs.com/docs/c/c", "name": "Title from yaml header c"}
       ]
     }
+---
```

Comment:
use xunit `InlineData` test to test the `GetStandardizedBookmark` helper method?

Reply:
Done.